src/Symfony/Component/PropertyInfo/PropertyInfoExtractor.php

```diff
@@ -103,13 +88,13 @@ public function isWritable($class, $property, array $context = array())
     /**
      * Iterates over registered extractors and return the first value found.
      *
-     * @param array  $extractors
-     * @param string $method
-     * @param array  $arguments
+     * @param array|\Traversable $extractors
```

Comment:
Yes, and we [already did](https://github.com/symfony/symfony/blob/b056d40fb3e990b81054311c54e186fe455aa79a/src/Symfony/Component/Security/Core/Authentication/AuthenticationProviderManager.php#L40) :)